src/main/resources/db/changelog/changes/v1.1.0/update-drv-loan-details-definition.xml

```diff
@@ -0,0 +1,352 @@
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+  <changeSet id="add-filter-value-getters-for-loan-details" author="mweaver@ebsco.com">
+    <preConditions onFail="CONTINUE">
+      <viewExists viewName="src_inventory_item"/>
+      <viewExists viewName="src_inventory_holdings_record"/>
+      <viewExists viewName="src_inventory_instance"/>
+      <viewExists viewName="src_inventory_material_type"/>
+      <viewExists viewName="src_inventory_location"/>
+      <viewExists viewName="src_inventory_service_point"/>
+      <viewExists viewName="src_inventory_loclibrary"/>
+      <viewExists viewName="src_inventory_call_number_type"/>
+      <viewExists viewName="src_circulation_loan"/>
+      <viewExists viewName="src_circulation_loan_policy"/>
+      <viewExists viewName="src_users_groups"/>
+      <viewExists viewName="src_users_users"/>
+      <sqlCheck expectedResult="2">
+        SELECT COUNT(*) FROM pg_matviews WHERE schemaname = '${tenant_id}_mod_fqm_manager'AND matviewname IN ('drv_circulation_loan_status', 'drv_inventory_item_status');
+      </sqlCheck>
+    </preConditions>
+
+    <comment>Add filterValueGetters to various columns</comment>
+    <update tableName="entity_type_definition">
+      <column name="definition">
+        {
+          "id": "4e09d89a-44ed-418e-a9cc-820dfb27bf3a",
+          "name": "drv_loan_details",
+          "private": false,
+          "fromClause": "src_circulation_loan LEFT JOIN src_circulation_loan_policy loan_policy_ref_data ON loan_policy_ref_data.id = ((src_circulation_loan.jsonb ->> 'loanPolicyId'::text)::uuid) LEFT JOIN src_inventory_service_point cospi ON cospi.id = \"left\"(lower(src_circulation_loan.jsonb ->> 'checkoutServicePointId'::text), 600)::uuid LEFT JOIN src_inventory_service_point cispi ON cispi.id = \"left\"(lower(src_circulation_loan.jsonb ->> 'checkinServicePointId'::text), 600)::uuid JOIN src_inventory_item item_details ON item_details.id = \"left\"(lower(f_unaccent(src_circulation_loan.jsonb ->> 'itemId'::text)), 600)::uuid LEFT JOIN src_inventory_material_type material_type_ref_data ON material_type_ref_data.id = item_details.materialtypeid JOIN src_users_users user_details ON user_details.id = \"left\"(lower(f_unaccent(src_circulation_loan.jsonb ->> 'userId'::text)), 600)::uuid LEFT JOIN src_users_groups patron_id_ref_data ON patron_id_ref_data.id = user_details.patrongroup JOIN src_inventory_holdings_record hrim ON item_details.holdingsrecordid = hrim.id JOIN src_inventory_instance instance_details ON hrim.instanceid = instance_details.id",
```

Comment:
I modified the join condition on `patron_id_ref_data`, to make it use `user_details.patrongroup` instead of `user_details.jsonb ->> 'patronGroup'`. Both versions of the patron group have indexes, but the `jsonb` one requires a function. Since it's a `join`, using the `jsonb` one could mean calling that function for every row in `user_details` in order to compare it to `patron_id_ref_data.id`, which could could expensive. Using the plain column is pretty much free